### DISCUSSION TITLE
Multi-editing not working

### DISCUSSION THREAD

**Discussion by ThomasDV1984** [opengisch/QField#5651] [Bugs]
Hi, 

We started a new project today and multi editing was not working in qfield. At first I tought is was a configuration problem but then we tried older project we where experiencing the simular behaviour. And in an other project we get this 
![Afbeelding van WhatsApp op 2024-09-16 om 11 27 16_ab3deb31](https://github.com/user-attachments/assets/7994a2b4-defe-4969-8391-a37029b764b9)
Here all the other attributes are gone.
This how it should look (when not multi editing.
![Afbeelding van WhatsApp op 2024-09-16 om 11 30 57_bf18f7ed](https://github.com/user-attachments/assets/bf1b4828-5a61-402e-9050-5e5a3e66fd87)

**Comment by nirvn** [comment]
@ThomasDV1984 , can you share a sample project? I can't replicate here:

![image](https://github.com/user-attachments/assets/b7c286a8-2a17-4c84-8e2c-55002fe514fc)

**Reply by ThomasDV1984** [reply]
Does the multi editing work within your project?

**Reply by ThomasDV1984** [reply]
In all our projects we can multi editing. But if you select 4 objects only in one object the values are changed.

**Reply by nirvn** [reply]
@ThomasDV1984 , ah, the 1 out of 4 features changed is fixed here (https://github.com/opengisch/QField/pull/5657). You can give the APKs in the PR a try to confirm it's fixed.

What I could not replicate is the broken feature form state your screenshot above demonstrates. I had assumed it was what wasn't working for you. :)

That being said, if you could share this old project that leads to a broken feature form our way, it'd be useful.

**Comment by ThomasDV1984** [comment]
"You can give the APKs in the PR a try " What does this mean?

**Reply by nirvn** [reply]
@ThomasDV1984 , you can go in the pull request (#5657) and download an APK (if you have a recent device it'll be arm64-android) in the comments section (here: https://github.com/opengisch/QField/pull/5657#issuecomment-2354487548). This will install a development version of QField on your system, which will allow you to confirm that multi-editing works for you.

**Reply by ThomasDV1984** [reply]
It seems to work

**Reply by nirvn** [reply]
@ThomasDV1984 , that seems good news! :wink: thanks for confirming. You can expect 3.4.2 in the coming days. Happy field mapping!

**Reply by ThomasDV1984** [reply]
Can we work i  the beta for now?

**Reply by nirvn** [reply]
@ThomasDV1984 , yes, you can, but QField and QField beta do not use the same user profile storage. If you work on cloud projects, make sure you push your changes before switching from one to the other.

If you are using exported projects for cable transfer, the location is slightly different (Android/data/ch.opengis.qfield/... vs Android/data/ch.opengis.qfield_beta/...).

**Reply by ThomasDV1984** [reply]
We are using a postgis db. Foto's are pushed to the cloud?

**Reply by nirvn** [reply]
@ThomasDV1984 , nice, then if you're handling your data through an online layer, that shouldn't be a problem. Thanks for reporting the regression.

**Reply by jmparries** [reply]
@nirvn I am having the same issue as @ThomasDV1984 with the multi editing. In my current project it is a feature that saves me a lot of time. So I am very much looking forward for the updated version. Thanks for the quick reactions here!
